config/validateuser_test.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v3"
+)
+
+func TestInvalidUser(t *testing.T) {
```

Comment:
not sure its mandatory, I have seen the name of the test function matches the function that's being tested.